bundles/core/src/test/resources/image/v2/exporter-image4.json

```diff
@@ -1,9 +1,9 @@
 {
-    "srcUriTemplate"        : "/core/content/test/_jcr_content/root/image4.coreimg{.width}.png/1494867377756.png",
+    "srcUriTemplate"        : "/core/content/test/_jcr_content/root/image4.coreimg{.width}.png/1494867377756/adobe-systems-logo-and-wordmark.png",
     "widths": [],
     "areas": [],
     "lazyEnabled"           : false,
     "title"                 : "Adobe Systems Logo and Wordmark",
-    "src"                   : "/core/content/test/_jcr_content/root/image4.coreimg.png/1494867377756.png",
+    "src"                   : "/core/content/test/_jcr_content/root/image4.coreimg.png/1494867377756/adobe-systems-logo-and-wordmark.png",
```

Comment:
@ompandeyy @richardhand It seems that all the tests have been updated to contain a filename in the URL. Do we still have tests without? Both use cases should be covered by tests.

Reply:
@jckautzmann - good point - I did test both scenarios manually but an automated test would be a good idea.

Reply:
@ompandeyy will you have the bandwidth to add a case the no name scenario? Following that I would recommend we merge here.

Reply:
@richardhand Unfortunately not today because of some unexpected Laptop failure in my Office. I can target this by Monday though.

Reply:
Great! Thanks again for your efforts

Reply:
@jckautzmann  @richardhand I have added tests for image and template type cases for the Servlet. I am not sure if we really need a test case for a missing name in the `fileReference `for `ImageImpl.java(v1 and v2)`